requirements.txt

```diff
@@ -1,3 +1,4 @@
+demquery>=0.2.0
```

Comment:
Place this dependency in `requirements-dev.txt` instead, as this is an optional dependency for non-core functionality (see previous comment above).